roles/volumes/tasks/main.yml

```
@@ -0,0 +1,9 @@
+---
+- name: Set a fact about the Ansible python interpreter
+  ansible.builtin.set_fact:
+    old_ansible_python_interpreter: "{{ ansible_python_interpreter | default('/usr/bin/python' + ansible_python.version.major | string) }}"
```

I dropped Python 2 support from other roles:
```suggestion
    old_ansible_python_interpreter: "{{ ansible_python_interpreter | default('/usr/bin/python3') }}"
```